### PR TITLE
Added back to top functionality to stache

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
A few suggestions:
- Use `$` instead of `jQuery`, which is safe since we're passing it in to our self-executing function.
- Move your `var` declarations to top `var` statement.
- Save `jQuery('.back-to-top')` in a variable since it's created several times
- Make `bttoffset` a `data-offset` attribute and read using `.data()` method.  In the markup, use a Stache variable for that value.

**Comment by blackbaud-brandonhare** [comment]
Made the changes you suggested. Thanks for reviewing it for me.

**Comment by Blackbaud-BobbyEarl** [comment]
Looks good @blackbaud-brandonhare .  I think there's still one more case of combining `$backToTop` in the top `var` block.

**Comment by blackbaud-brandonhare** [comment]
Oh duh. Yeah, I’ll fix that. Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
Once last thing... let's add a check for `$backToTop` before adding our listener to the scroll event.  No point in adding that if people haven't turned the Stache variable on.

**Comment by blackbaud-brandonhare** [comment]
Done!

File: src/js/stache-jquery.js [hunk 1]
```diff
@@ -107,5 +110,22 @@
             }
         }
     });
-
+  
+  //Back-to-top
+  // Code from here: http://www.developerdrive.com/2013/07/using-jquery-to-add-a-dynamic-back-to-top-floating-button-with-smooth-scroll/
+   $(window).scroll(function() {
```
Comment (Blackbaud-BobbyEarl) [inline src/js/stache-jquery.js]: Move this into your if statement.

Comment (blackbaud-brandonhare) [inline src/js/stache-jquery.js]: Fixed it. 8th time's the charm. :-P

**Pull request merged**